### PR TITLE
Adding perfmon test to avocado[v2]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: praveen@linux.vnet.ibm.com
added perfmon Test case

v1 #118 

```

v2: remove duplicate code
v2: change condition `not` to` or`
v2: make self.fail when test got failed
```

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
please see run log in below link 

https://paste.fedoraproject.org/400716/70233472/

File: perf/perfmon.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    performance monitoring on Linux : test  perf_events on Linux
+    """
+
+    def setUp(self):
+
+        # Check for basic utilities
+        smm = SoftwareManager()
+
+        for package in ("gcc", "make"):
+            if not (smm.check_installed(package) or smm.install(package)):
```
Comment (apahim) [inline perf/perfmon.py]: Hi @PraveenPenguin, sorry about my confusion. It was correct before. Please change it back to:

```
if not smm.check_installed(package) and not smm.install(package):
```

Those are the conditions that will fail the test, not to go ahead (as it was my first impression).

**Comment by apahim** [comment]
@PraveenPenguinm just a minor comment (my fault actually). You can amend the commit and run `$ git push -f` (instead of sending a new pull request).

File: perf/perfmon.py (hunk 1 above)
Comment (PraveenPenguin) [inline perf/perfmon.py]: sure will do thanks

**Comment by PraveenPenguin** [comment]
@apahim  made changes and already pushed the change , Please have a look 

Thanks

**Comment by apahim** [comment]
lgtm

**Pull request merged**